core/iwasm/common/gc/gc_common.c

```diff
@@ -244,17 +244,32 @@ wasm_ref_type_equal(const wasm_ref_type_t *ref_type1,
                     const wasm_ref_type_t *ref_type2,
```

Comment:
@wenyongh here when calling `wasm_ref_type_normalize`, the content of `wasm_ref_type_t ` might be changed, should we 1) create a new pointer in `wasm_ref_type_normalize`  or 2）remove `const` modifier in `wasm_ref_type_equal`?

Reply:
Had better define another ref_type for internal use: e.g. `wasm_ref_type_t ref_type1_norm = wasm_ref_type_normalize(ref_type1);`